packages/lodestar/src/api/rest/routes/validator/syncing.ts

```diff
@@ -0,0 +1,17 @@
+import { IFastifyServer } from "../..";
+import { IApiModules } from "../../..";
+import { IncomingMessage, ServerResponse } from 'http';
+
+
+export const registerSyncingMiddleware = (fastify: IFastifyServer, modules: IApiModules): void => {
+  const syncingMiddleware = (request: IncomingMessage, response: ServerResponse, next: Function) => {
+    if (!modules.sync.isSynced()) {
```

Comment:
I think this method is async. If middleware cannot support async stuff, we can put this PR on hold until we figure out how to put some prop in sync that would get updated peridically

Reply:
asyn middleware is quite common in Express. Fastify also mentioned that "Fastify out of the box provides an asynchronous middleware engine compatible with Express and Restify middlewares."